CRITICAL - 2022-04-15 09:34:58 --> Undefined property: App\Controllers\Login::$input
#0 C:\xampp\htdocs\codeIgniter\new-app2\app\Controllers\Login.php(245): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined prope...', 'C:\\xampp\\htdocs...', 245, Array)
#1 C:\xampp\htdocs\codeIgniter\new-app2\system\CodeIgniter.php(824): App\Controllers\Login->login_verify()
#2 C:\xampp\htdocs\codeIgniter\new-app2\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#3 C:\xampp\htdocs\codeIgniter\new-app2\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\codeIgniter\new-app2\index.php(37): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2022-04-15 09:36:03 --> Undefined property: App\Controllers\Login::$input
#0 C:\xampp\htdocs\codeIgniter\new-app2\app\Controllers\Login.php(245): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined prope...', 'C:\\xampp\\htdocs...', 245, Array)
#1 C:\xampp\htdocs\codeIgniter\new-app2\system\CodeIgniter.php(824): App\Controllers\Login->login_verify()
#2 C:\xampp\htdocs\codeIgniter\new-app2\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#3 C:\xampp\htdocs\codeIgniter\new-app2\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\codeIgniter\new-app2\index.php(37): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2022-04-15 09:36:22 --> Call to undefined method CodeIgniter\HTTP\IncomingRequest::post()
#0 C:\xampp\htdocs\codeIgniter\new-app2\system\CodeIgniter.php(824): App\Controllers\Login->login_verify()
#1 C:\xampp\htdocs\codeIgniter\new-app2\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#2 C:\xampp\htdocs\codeIgniter\new-app2\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#3 C:\xampp\htdocs\codeIgniter\new-app2\index.php(37): CodeIgniter\CodeIgniter->run()
#4 {main}
CRITICAL - 2022-04-15 10:28:11 --> The action you requested is not allowed.
#0 C:\xampp\htdocs\codeIgniter\new-app2\system\Security\Security.php(277): CodeIgniter\Security\Exceptions\SecurityException::forDisallowedAction()
#1 C:\xampp\htdocs\codeIgniter\new-app2\system\Filters\CSRF.php(53): CodeIgniter\Security\Security->verify(Object(CodeIgniter\HTTP\IncomingRequest))
#2 C:\xampp\htdocs\codeIgniter\new-app2\system\Filters\Filters.php(173): CodeIgniter\Filters\CSRF->before(Object(CodeIgniter\HTTP\IncomingRequest), NULL)
#3 C:\xampp\htdocs\codeIgniter\new-app2\system\CodeIgniter.php(384): CodeIgniter\Filters\Filters->run('user_emailCheck', 'before')
#4 C:\xampp\htdocs\codeIgniter\new-app2\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\codeIgniter\new-app2\index.php(37): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2022-04-15 10:28:13 --> The action you requested is not allowed.
#0 C:\xampp\htdocs\codeIgniter\new-app2\system\Security\Security.php(277): CodeIgniter\Security\Exceptions\SecurityException::forDisallowedAction()
#1 C:\xampp\htdocs\codeIgniter\new-app2\system\Filters\CSRF.php(53): CodeIgniter\Security\Security->verify(Object(CodeIgniter\HTTP\IncomingRequest))
#2 C:\xampp\htdocs\codeIgniter\new-app2\system\Filters\Filters.php(173): CodeIgniter\Filters\CSRF->before(Object(CodeIgniter\HTTP\IncomingRequest), NULL)
#3 C:\xampp\htdocs\codeIgniter\new-app2\system\CodeIgniter.php(384): CodeIgniter\Filters\Filters->run('user_phoneCheck', 'before')
#4 C:\xampp\htdocs\codeIgniter\new-app2\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\codeIgniter\new-app2\index.php(37): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2022-04-15 10:33:44 --> The action you requested is not allowed.
#0 C:\xampp\htdocs\codeIgniter\new-app2\system\Security\Security.php(277): CodeIgniter\Security\Exceptions\SecurityException::forDisallowedAction()
#1 C:\xampp\htdocs\codeIgniter\new-app2\system\Filters\CSRF.php(53): CodeIgniter\Security\Security->verify(Object(CodeIgniter\HTTP\IncomingRequest))
#2 C:\xampp\htdocs\codeIgniter\new-app2\system\Filters\Filters.php(173): CodeIgniter\Filters\CSRF->before(Object(CodeIgniter\HTTP\IncomingRequest), NULL)
#3 C:\xampp\htdocs\codeIgniter\new-app2\system\CodeIgniter.php(384): CodeIgniter\Filters\Filters->run('user_emailCheck', 'before')
#4 C:\xampp\htdocs\codeIgniter\new-app2\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\codeIgniter\new-app2\index.php(37): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2022-04-15 10:33:47 --> The action you requested is not allowed.
#0 C:\xampp\htdocs\codeIgniter\new-app2\system\Security\Security.php(277): CodeIgniter\Security\Exceptions\SecurityException::forDisallowedAction()
#1 C:\xampp\htdocs\codeIgniter\new-app2\system\Filters\CSRF.php(53): CodeIgniter\Security\Security->verify(Object(CodeIgniter\HTTP\IncomingRequest))
#2 C:\xampp\htdocs\codeIgniter\new-app2\system\Filters\Filters.php(173): CodeIgniter\Filters\CSRF->before(Object(CodeIgniter\HTTP\IncomingRequest), NULL)
#3 C:\xampp\htdocs\codeIgniter\new-app2\system\CodeIgniter.php(384): CodeIgniter\Filters\Filters->run('user_phoneCheck', 'before')
#4 C:\xampp\htdocs\codeIgniter\new-app2\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\codeIgniter\new-app2\index.php(37): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2022-04-15 10:38:42 --> Class 'App\Controllers\Time' not found
#0 C:\xampp\htdocs\codeIgniter\new-app2\system\CodeIgniter.php(824): App\Controllers\Login->login_verify()
#1 C:\xampp\htdocs\codeIgniter\new-app2\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#2 C:\xampp\htdocs\codeIgniter\new-app2\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#3 C:\xampp\htdocs\codeIgniter\new-app2\index.php(37): CodeIgniter\CodeIgniter->run()
#4 {main}
ERROR - 2022-04-15 10:39:37 --> Unknown column 'updated_at' in 'field list'
CRITICAL - 2022-04-15 10:39:37 --> Unknown column 'updated_at' in 'field list'
#0 C:\xampp\htdocs\codeIgniter\new-app2\system\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `us...', 0)
#1 C:\xampp\htdocs\codeIgniter\new-app2\system\Database\BaseConnection.php(670): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\xampp\htdocs\codeIgniter\new-app2\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\xampp\htdocs\codeIgniter\new-app2\system\Database\BaseBuilder.php(1759): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\htdocs\codeIgniter\new-app2\system\Model.php(239): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\codeIgniter\new-app2\system\BaseModel.php(738): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\codeIgniter\new-app2\system\Model.php(620): CodeIgniter\BaseModel->insert(Array, false)
#7 C:\xampp\htdocs\codeIgniter\new-app2\system\BaseModel.php(658): CodeIgniter\Model->insert(Array, false)
#8 C:\xampp\htdocs\codeIgniter\new-app2\app\Controllers\Login.php(297): CodeIgniter\BaseModel->save(Array)
#9 C:\xampp\htdocs\codeIgniter\new-app2\system\CodeIgniter.php(824): App\Controllers\Login->login_verify()
#10 C:\xampp\htdocs\codeIgniter\new-app2\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#11 C:\xampp\htdocs\codeIgniter\new-app2\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\codeIgniter\new-app2\index.php(37): CodeIgniter\CodeIgniter->run()
#13 {main}
CRITICAL - 2022-04-15 10:42:57 --> syntax error, unexpected 'exit' (T_EXIT), expecting ';' or ','
#0 C:\xampp\htdocs\codeIgniter\new-app2\system\Autoloader\Autoloader.php(241): CodeIgniter\Autoloader\Autoloader->includeFile('C:\\xampp\\htdocs...')
#1 C:\xampp\htdocs\codeIgniter\new-app2\system\Autoloader\Autoloader.php(219): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers...')
#2 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers...')
#3 [internal function]: spl_autoload_call('App\\Controllers...')
#4 C:\xampp\htdocs\codeIgniter\new-app2\system\CodeIgniter.php(789): class_exists('\\App\\Controller...', true)
#5 C:\xampp\htdocs\codeIgniter\new-app2\system\CodeIgniter.php(397): CodeIgniter\CodeIgniter->startController()
#6 C:\xampp\htdocs\codeIgniter\new-app2\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\codeIgniter\new-app2\index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2022-04-15 10:43:10 --> syntax error, unexpected 'exit' (T_EXIT), expecting ';' or ','
#0 C:\xampp\htdocs\codeIgniter\new-app2\system\Autoloader\Autoloader.php(241): CodeIgniter\Autoloader\Autoloader->includeFile('C:\\xampp\\htdocs...')
#1 C:\xampp\htdocs\codeIgniter\new-app2\system\Autoloader\Autoloader.php(219): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers...')
#2 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers...')
#3 [internal function]: spl_autoload_call('App\\Controllers...')
#4 C:\xampp\htdocs\codeIgniter\new-app2\system\CodeIgniter.php(789): class_exists('\\App\\Controller...', true)
#5 C:\xampp\htdocs\codeIgniter\new-app2\system\CodeIgniter.php(397): CodeIgniter\CodeIgniter->startController()
#6 C:\xampp\htdocs\codeIgniter\new-app2\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\codeIgniter\new-app2\index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2022-04-15 10:43:23 --> Undefined offset: 0
#0 C:\xampp\htdocs\codeIgniter\new-app2\app\Controllers\Login.php(305): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined offse...', 'C:\\xampp\\htdocs...', 305, Array)
#1 C:\xampp\htdocs\codeIgniter\new-app2\system\CodeIgniter.php(824): App\Controllers\Login->login_verify()
#2 C:\xampp\htdocs\codeIgniter\new-app2\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#3 C:\xampp\htdocs\codeIgniter\new-app2\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\codeIgniter\new-app2\index.php(37): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2022-04-15 10:43:56 --> Undefined offset: 0
#0 C:\xampp\htdocs\codeIgniter\new-app2\app\Controllers\Login.php(305): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined offse...', 'C:\\xampp\\htdocs...', 305, Array)
#1 C:\xampp\htdocs\codeIgniter\new-app2\system\CodeIgniter.php(824): App\Controllers\Login->login_verify()
#2 C:\xampp\htdocs\codeIgniter\new-app2\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#3 C:\xampp\htdocs\codeIgniter\new-app2\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\codeIgniter\new-app2\index.php(37): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2022-04-15 21:29:39 --> The action you requested is not allowed.
#0 C:\xampp\htdocs\codeIgniter\new-app2\system\Security\Security.php(277): CodeIgniter\Security\Exceptions\SecurityException::forDisallowedAction()
#1 C:\xampp\htdocs\codeIgniter\new-app2\system\Filters\CSRF.php(53): CodeIgniter\Security\Security->verify(Object(CodeIgniter\HTTP\IncomingRequest))
#2 C:\xampp\htdocs\codeIgniter\new-app2\system\Filters\Filters.php(173): CodeIgniter\Filters\CSRF->before(Object(CodeIgniter\HTTP\IncomingRequest), NULL)
#3 C:\xampp\htdocs\codeIgniter\new-app2\system\CodeIgniter.php(384): CodeIgniter\Filters\Filters->run('user/user_email...', 'before')
#4 C:\xampp\htdocs\codeIgniter\new-app2\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\codeIgniter\new-app2\index.php(37): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2022-04-15 21:29:49 --> The action you requested is not allowed.
#0 C:\xampp\htdocs\codeIgniter\new-app2\system\Security\Security.php(277): CodeIgniter\Security\Exceptions\SecurityException::forDisallowedAction()
#1 C:\xampp\htdocs\codeIgniter\new-app2\system\Filters\CSRF.php(53): CodeIgniter\Security\Security->verify(Object(CodeIgniter\HTTP\IncomingRequest))
#2 C:\xampp\htdocs\codeIgniter\new-app2\system\Filters\Filters.php(173): CodeIgniter\Filters\CSRF->before(Object(CodeIgniter\HTTP\IncomingRequest), NULL)
#3 C:\xampp\htdocs\codeIgniter\new-app2\system\CodeIgniter.php(384): CodeIgniter\Filters\Filters->run('user/user_email...', 'before')
#4 C:\xampp\htdocs\codeIgniter\new-app2\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\codeIgniter\new-app2\index.php(37): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2022-04-15 21:32:14 --> The action you requested is not allowed.
#0 C:\xampp\htdocs\codeIgniter\new-app2\system\Security\Security.php(277): CodeIgniter\Security\Exceptions\SecurityException::forDisallowedAction()
#1 C:\xampp\htdocs\codeIgniter\new-app2\system\Filters\CSRF.php(53): CodeIgniter\Security\Security->verify(Object(CodeIgniter\HTTP\IncomingRequest))
#2 C:\xampp\htdocs\codeIgniter\new-app2\system\Filters\Filters.php(173): CodeIgniter\Filters\CSRF->before(Object(CodeIgniter\HTTP\IncomingRequest), NULL)
#3 C:\xampp\htdocs\codeIgniter\new-app2\system\CodeIgniter.php(384): CodeIgniter\Filters\Filters->run('user/user_email...', 'before')
#4 C:\xampp\htdocs\codeIgniter\new-app2\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\codeIgniter\new-app2\index.php(37): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2022-04-15 21:47:11 --> The action you requested is not allowed.
#0 C:\xampp\htdocs\codeIgniter\new-app2\system\Security\Security.php(277): CodeIgniter\Security\Exceptions\SecurityException::forDisallowedAction()
#1 C:\xampp\htdocs\codeIgniter\new-app2\system\Filters\CSRF.php(53): CodeIgniter\Security\Security->verify(Object(CodeIgniter\HTTP\IncomingRequest))
#2 C:\xampp\htdocs\codeIgniter\new-app2\system\Filters\Filters.php(173): CodeIgniter\Filters\CSRF->before(Object(CodeIgniter\HTTP\IncomingRequest), NULL)
#3 C:\xampp\htdocs\codeIgniter\new-app2\system\CodeIgniter.php(384): CodeIgniter\Filters\Filters->run('user/user_phone...', 'before')
#4 C:\xampp\htdocs\codeIgniter\new-app2\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\codeIgniter\new-app2\index.php(37): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2022-04-15 21:52:12 --> The action you requested is not allowed.
#0 C:\xampp\htdocs\codeIgniter\new-app2\system\Security\Security.php(277): CodeIgniter\Security\Exceptions\SecurityException::forDisallowedAction()
#1 C:\xampp\htdocs\codeIgniter\new-app2\system\Filters\CSRF.php(53): CodeIgniter\Security\Security->verify(Object(CodeIgniter\HTTP\IncomingRequest))
#2 C:\xampp\htdocs\codeIgniter\new-app2\system\Filters\Filters.php(173): CodeIgniter\Filters\CSRF->before(Object(CodeIgniter\HTTP\IncomingRequest), NULL)
#3 C:\xampp\htdocs\codeIgniter\new-app2\system\CodeIgniter.php(384): CodeIgniter\Filters\Filters->run('user/user_phone...', 'before')
#4 C:\xampp\htdocs\codeIgniter\new-app2\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\codeIgniter\new-app2\index.php(37): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2022-04-15 21:52:33 --> The action you requested is not allowed.
#0 C:\xampp\htdocs\codeIgniter\new-app2\system\Security\Security.php(277): CodeIgniter\Security\Exceptions\SecurityException::forDisallowedAction()
#1 C:\xampp\htdocs\codeIgniter\new-app2\system\Filters\CSRF.php(53): CodeIgniter\Security\Security->verify(Object(CodeIgniter\HTTP\IncomingRequest))
#2 C:\xampp\htdocs\codeIgniter\new-app2\system\Filters\Filters.php(173): CodeIgniter\Filters\CSRF->before(Object(CodeIgniter\HTTP\IncomingRequest), NULL)
#3 C:\xampp\htdocs\codeIgniter\new-app2\system\CodeIgniter.php(384): CodeIgniter\Filters\Filters->run('user/user_email...', 'before')
#4 C:\xampp\htdocs\codeIgniter\new-app2\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\codeIgniter\new-app2\index.php(37): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2022-04-15 21:53:39 --> The action you requested is not allowed.
#0 C:\xampp\htdocs\codeIgniter\new-app2\system\Security\Security.php(277): CodeIgniter\Security\Exceptions\SecurityException::forDisallowedAction()
#1 C:\xampp\htdocs\codeIgniter\new-app2\system\Filters\CSRF.php(53): CodeIgniter\Security\Security->verify(Object(CodeIgniter\HTTP\IncomingRequest))
#2 C:\xampp\htdocs\codeIgniter\new-app2\system\Filters\Filters.php(173): CodeIgniter\Filters\CSRF->before(Object(CodeIgniter\HTTP\IncomingRequest), NULL)
#3 C:\xampp\htdocs\codeIgniter\new-app2\system\CodeIgniter.php(384): CodeIgniter\Filters\Filters->run('user/user_phone...', 'before')
#4 C:\xampp\htdocs\codeIgniter\new-app2\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\codeIgniter\new-app2\index.php(37): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2022-04-15 21:56:38 --> The action you requested is not allowed.
#0 C:\xampp\htdocs\codeIgniter\new-app2\system\Security\Security.php(277): CodeIgniter\Security\Exceptions\SecurityException::forDisallowedAction()
#1 C:\xampp\htdocs\codeIgniter\new-app2\system\Filters\CSRF.php(53): CodeIgniter\Security\Security->verify(Object(CodeIgniter\HTTP\IncomingRequest))
#2 C:\xampp\htdocs\codeIgniter\new-app2\system\Filters\Filters.php(173): CodeIgniter\Filters\CSRF->before(Object(CodeIgniter\HTTP\IncomingRequest), NULL)
#3 C:\xampp\htdocs\codeIgniter\new-app2\system\CodeIgniter.php(384): CodeIgniter\Filters\Filters->run('user/user_email...', 'before')
#4 C:\xampp\htdocs\codeIgniter\new-app2\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\codeIgniter\new-app2\index.php(37): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2022-04-15 21:58:04 --> The action you requested is not allowed.
#0 C:\xampp\htdocs\codeIgniter\new-app2\system\Security\Security.php(277): CodeIgniter\Security\Exceptions\SecurityException::forDisallowedAction()
#1 C:\xampp\htdocs\codeIgniter\new-app2\system\Filters\CSRF.php(53): CodeIgniter\Security\Security->verify(Object(CodeIgniter\HTTP\IncomingRequest))
#2 C:\xampp\htdocs\codeIgniter\new-app2\system\Filters\Filters.php(173): CodeIgniter\Filters\CSRF->before(Object(CodeIgniter\HTTP\IncomingRequest), NULL)
#3 C:\xampp\htdocs\codeIgniter\new-app2\system\CodeIgniter.php(384): CodeIgniter\Filters\Filters->run('user/user_email...', 'before')
#4 C:\xampp\htdocs\codeIgniter\new-app2\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\codeIgniter\new-app2\index.php(37): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2022-04-15 22:01:19 --> The action you requested is not allowed.
#0 C:\xampp\htdocs\codeIgniter\new-app2\system\Security\Security.php(277): CodeIgniter\Security\Exceptions\SecurityException::forDisallowedAction()
#1 C:\xampp\htdocs\codeIgniter\new-app2\system\Filters\CSRF.php(53): CodeIgniter\Security\Security->verify(Object(CodeIgniter\HTTP\IncomingRequest))
#2 C:\xampp\htdocs\codeIgniter\new-app2\system\Filters\Filters.php(173): CodeIgniter\Filters\CSRF->before(Object(CodeIgniter\HTTP\IncomingRequest), NULL)
#3 C:\xampp\htdocs\codeIgniter\new-app2\system\CodeIgniter.php(384): CodeIgniter\Filters\Filters->run('user/user_phone...', 'before')
#4 C:\xampp\htdocs\codeIgniter\new-app2\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\codeIgniter\new-app2\index.php(37): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2022-04-15 23:12:24 --> Undefined property: App\Filters\LoggedIn::$session
#0 C:\xampp\htdocs\codeIgniter\new-app2\app\Filters\LoggedIn.php(29): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined prope...', 'C:\\xampp\\htdocs...', 29, Array)
#1 C:\xampp\htdocs\codeIgniter\new-app2\system\Filters\Filters.php(173): App\Filters\LoggedIn->before(Object(CodeIgniter\HTTP\IncomingRequest), NULL)
#2 C:\xampp\htdocs\codeIgniter\new-app2\system\CodeIgniter.php(384): CodeIgniter\Filters\Filters->run('user_panel/dash...', 'before')
#3 C:\xampp\htdocs\codeIgniter\new-app2\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\codeIgniter\new-app2\index.php(37): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2022-04-15 23:13:24 --> Undefined property: App\Filters\LoggedIn::$session
#0 C:\xampp\htdocs\codeIgniter\new-app2\app\Filters\LoggedIn.php(29): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined prope...', 'C:\\xampp\\htdocs...', 29, Array)
#1 C:\xampp\htdocs\codeIgniter\new-app2\system\Filters\Filters.php(173): App\Filters\LoggedIn->before(Object(CodeIgniter\HTTP\IncomingRequest), NULL)
#2 C:\xampp\htdocs\codeIgniter\new-app2\system\CodeIgniter.php(384): CodeIgniter\Filters\Filters->run('user_panel/dash...', 'before')
#3 C:\xampp\htdocs\codeIgniter\new-app2\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\codeIgniter\new-app2\index.php(37): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2022-04-15 23:17:39 --> Undefined property: App\Controllers\UserController::$login_user
#0 C:\xampp\htdocs\codeIgniter\new-app2\app\Controllers\UserController.php(18): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined prope...', 'C:\\xampp\\htdocs...', 18, Array)
#1 C:\xampp\htdocs\codeIgniter\new-app2\system\CodeIgniter.php(824): App\Controllers\UserController->logout()
#2 C:\xampp\htdocs\codeIgniter\new-app2\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserController))
#3 C:\xampp\htdocs\codeIgniter\new-app2\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\codeIgniter\new-app2\index.php(37): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2022-04-15 23:18:23 --> Trying to access array offset on value of type null
#0 C:\xampp\htdocs\codeIgniter\new-app2\app\Controllers\UserController.php(22): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Trying to acces...', 'C:\\xampp\\htdocs...', 22, Array)
#1 C:\xampp\htdocs\codeIgniter\new-app2\system\CodeIgniter.php(824): App\Controllers\UserController->logout()
#2 C:\xampp\htdocs\codeIgniter\new-app2\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserController))
#3 C:\xampp\htdocs\codeIgniter\new-app2\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\codeIgniter\new-app2\index.php(37): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2022-04-15 23:19:14 --> Undefined index: loginTBL_Id
#0 C:\xampp\htdocs\codeIgniter\new-app2\app\Controllers\UserController.php(22): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined index...', 'C:\\xampp\\htdocs...', 22, Array)
#1 C:\xampp\htdocs\codeIgniter\new-app2\system\CodeIgniter.php(824): App\Controllers\UserController->logout()
#2 C:\xampp\htdocs\codeIgniter\new-app2\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserController))
#3 C:\xampp\htdocs\codeIgniter\new-app2\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\codeIgniter\new-app2\index.php(37): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2022-04-15 23:19:34 --> Undefined property: App\Controllers\UserController::$usersModel
#0 C:\xampp\htdocs\codeIgniter\new-app2\app\Controllers\UserController.php(25): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined prope...', 'C:\\xampp\\htdocs...', 25, Array)
#1 C:\xampp\htdocs\codeIgniter\new-app2\system\CodeIgniter.php(824): App\Controllers\UserController->logout()
#2 C:\xampp\htdocs\codeIgniter\new-app2\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserController))
#3 C:\xampp\htdocs\codeIgniter\new-app2\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\codeIgniter\new-app2\index.php(37): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2022-04-15 23:19:53 --> Undefined property: App\Controllers\UserController::$usersModel
#0 C:\xampp\htdocs\codeIgniter\new-app2\app\Controllers\UserController.php(25): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined prope...', 'C:\\xampp\\htdocs...', 25, Array)
#1 C:\xampp\htdocs\codeIgniter\new-app2\system\CodeIgniter.php(824): App\Controllers\UserController->logout()
#2 C:\xampp\htdocs\codeIgniter\new-app2\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserController))
#3 C:\xampp\htdocs\codeIgniter\new-app2\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\codeIgniter\new-app2\index.php(37): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2022-04-15 23:20:19 --> Undefined index: user_data
#0 C:\xampp\htdocs\codeIgniter\new-app2\app\Controllers\UserController.php(25): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined index...', 'C:\\xampp\\htdocs...', 25, Array)
#1 C:\xampp\htdocs\codeIgniter\new-app2\system\CodeIgniter.php(824): App\Controllers\UserController->logout()
#2 C:\xampp\htdocs\codeIgniter\new-app2\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserController))
#3 C:\xampp\htdocs\codeIgniter\new-app2\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\codeIgniter\new-app2\index.php(37): CodeIgniter\CodeIgniter->run()
#5 {main}
